modules/op-release-notes-1-5.adoc

```diff
@@ -0,0 +1,317 @@
+
+* Support for optional workspaces are added to the `start` command.
+
+* The plugins will be looked up in the current path, if they are not present in the `plugins` directory.
+
+* The `tkn start [task | clustertask | pipeline]` command will start interactively and ask for the `params` value, even when you specify the default parameters are specified. To stop the interactive prompts, pass the `--use-param-defaults` flag at the time of invoking the command. For example:
++
+[source,terminal,subs="attributes+"]
+----
+$ tkn pipeline start build-and-deploy \
+    -w name=shared-workspace,volumeClaimTemplateFile=https://raw.githubusercontent.com/openshift/pipelines-tutorial/{pipelines-ver}/01_pipeline/03_persistent_volume_claim.yaml \
+    -p deployment-name=pipelines-vote-api \
+    -p git-url=https://github.com/openshift/pipelines-vote-api.git \
+    -p IMAGE=image-registry.openshift-image-registry.svc:5000/pipelines-tutorial/pipelines-vote-api \
+    --use-param-defaults
+----
+
+* The `version` field is added in the `tkn task describe` command.
+
+* The option to automatically select resources such as `TriggerTemplate`, or `TriggerBinding`, or `ClusterTriggerBinding`, or `Eventlistener`, has been added in the `describe` command, if only one is present.
+
+* In the `tkn pr describe` command, a section for skipped tasks is added.
+
+* Support for the `tkn clustertask logs` has been added.
+
+* The YAML merge and variable from `config.yaml` has been removed. In addition, the `release.yaml` file can now be more easily consumed by tools such as `kustomize` and `ytt`.
+
+* The support for resource names to contain the dot character (".") has been added.
+
+* The `hostAliases` array in the `PodTemplate` specification has been added to the pod-level override of hostname resolution. It has been achieved by modifying the `/etc/hosts` file.
+
+* A variable `$(tasks.status)` is introduced to access the aggregate execution status of tasks.
+
+* An entry-point binary build for Windows is added.
+
+
+[id="deprecated-features-1-5_{context}"]
+== Deprecated features
+
+* In the `when` expressions, support for fields written is PascalCase is removed. The `when` expressions only support fields written in lowercase.
++
+[NOTE]
+====
+If you had applied a pipeline with `when` expressions in Tekton Pipelines `v0.16` (Operator `v1.2.x`), you have to reapply it.
+====
+
+* When you upgrade the {pipelines-title} Operator to `v1.5`, the `openshift-client` and the `openshift-client-v-1-5-0` cluster tasks will have the `SCRIPT` parameter. However, the `ARGS` parameter and the `git` resource are removed from the specification of the `openshift-client` cluster task. This is a breaking change, and only those cluster tasks that do not have a specific version in the `name` field of the `ClusterTask` resource will seamlessly upgrade.
```

Comment:
To avoid future tense, consider rephrasing to, `When you upgrade the Red Hat OpenShift Pipelines Operator, the client and cluster tasks have the script parameter....cluster tasks that do not have a specific version in the name filed of the ClusterTaks resource upgrade.`